Classes/Command/Input/Normalizer/LocallangFileNameNormalizer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package friendsoftypo3/kickstarter.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace FriendsOfTYPO3\Kickstarter\Command\Input\Normalizer;
+
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+
+#[AutoconfigureTag('ext-kickstarter.inputHandler.locallang_file_name')]
+class LocallangFileNameNormalizer implements NormalizerInterface
+{
+    public function __invoke(?string $userInput): string
+    {
+        if ($userInput === null || $userInput === '') {
+            return 'locallang.xlf';
+        }
+
+        $cleanedUserInput = strtolower($userInput);
```

Comment:
Do we need that? Sometimes I work with ExtConf.xlf. Why should that not be allowed?

Reply:
done